patches/minecraft/net/minecraft/item/ItemStack.java.patch

```diff
@@ -214,15 +214,18 @@
        return list;
     }
  
-@@ -817,7 +841,7 @@
+@@ -817,9 +841,9 @@
              }
           }
        } else {
 -         multimap = this.func_77973_b().func_111205_h(p_111283_1_);
 +         multimap = this.func_77973_b().getAttributeModifiers(p_111283_1_, this);
        }
- 
+-
```

Comment:
Something got a little weird in the patch here. I don't recall seeing it in my PR because I was looking at a patch of a patch.

If I am reading this right, it is removing an empty line between `}` and `multimap = ...`?

Reply:
Yeah, that's what that is, Patch "`+ -`" is "this line was removed in this commit"